test/integration/support/poltergeist_onload_extensions.js

```diff
@@ -1,8 +1,9 @@
-var disableAnimationStyles = '-webkit-transition: none !important;' +
```

Comment:
please dont change any of the files that are not under eslint control.

Reply:
Ah yeah that must be a carry-over. I'll get that removed.